### ISSUE TITLE
Crash on Tapatalk launch

### ISSUE THREAD

**Issue by kvic-z** [kvic-z/pixelserv-tls#7]
**Client**

Tapatalk Android client. 

**Symptom**

On launch, Tapatalk sends a big chunk ~16KB of data using POST. Captured request looks like below. pixelserv-tls v2.0.1-rc2 crashes.

```
POST /v2.9/469687153111700/activities?access_token=&format=json&sdk=android HTTP/1.1
User-Agent: FBAndroidSDK.4.23.0
Accept-Language: es_ES
Content-Type: multipart/form-data; boundary=3i2ndDfv2rTHiSisAbouNdArYfORhtTPEefj3q2f
Transfer-Encoding: chunked
Host: graph.facebook.com
Connection: Keep-Alive
Accept-Encoding: gzip

7056
--3i2ndDfv2rTHiSisAbouNdArYfORhtTPEefj3q2f
Content-Disposition: form-data; name="format"

json
--3i2ndDfv2rTHiSisAbouNdArYfORhtTPEefj3q2f
Content-Disposition: form-data; name="sdk"

android
--3i2ndDfv2rTHiSisAbouNdArYfORhtTPEefj3q2f
Content-Disposition: form-data; name="custom_events_file"; filename="custom_events_file"
Content-Type: content/unknown

[{"_eventName":"Start_Session","_logTime":1512749547,"_ui":"null","_session_id":"5d35bdf4-4862-492e-b8fc-f80c574721c2","Channel":"Push_Conv"},{"_eventName":"fb_mobile_activate_app","_logTime":1512749256,"_ui":"CreateMessageActivity","_session_id":"5d35bdf4-4862-492e-b8fc-f80c574721c2","fb_mobile_launch_source":"Unclassified"},
```

**Comment by kvic-z** [comment]
**Analysis**

Tapatalk uses multi-part POST to upload contents to pixelserv-tls (i.e. ad servers). The request is not standard compliant. It doesn't contain "Content-Length." Hence, pixelserv-tls crashes.

**Comment by kvic-z** [comment]
Fixed crash with [0063f766](https://github.com/kvic-z/pixelserv-tls/commit/0063f766c5ded3e56c536b325275be7cb48d3bd3)

Full support of Transport-Encoding: chunked need more work and has to wait in a future release.
